crates/db_schema/src/source/oauth_provider.rs

```diff
@@ -86,7 +86,7 @@ impl Serialize for PublicOAuthProvider {
   }
 }
 
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, Serialize, Deserialize)]
```

Comment:
This struct isnt used anywhere in the api so these derives are not needed.

Reply:
I ended up needing the derives for `skip_serializing_none` to work.

Reply:
There is no use for skip_serializing_none if the struct is not exposed in the api.

Reply:
It looks like its returned in the GetSiteResponse, so it does need it.

Reply:
It only uses OAuthProvider and  PublicOAuthProvider, but Im talking about OAuthProviderInsertForm and OAuthProviderUpdateForm which dont need serde derives.

Reply:
Are the update and insert form even supposed to be exposed to the API client? If not, I'll remove the ts export from them.

Reply:
No they are only used internally in Rust.

Reply:
Removed I removed the TS export stuff from `OAuthProviderInsertForm` and `OAuthProviderUpdateForm`.